.gitignore

```diff
@@ -7,4 +7,5 @@
 .DS_Store
 npm-debug.log*
 .pnp.*
-/.meta
\ No newline at end of file
+/.meta
+.prettierrc
```

Comment:
Why?

Reply:
was a file of the pretty printer. dont want that on github

Reply:
that's the prettier configuration, it is needed

Reply:
fixed